porfolio.html

```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
+                        
+                    </article>  
+                    <div class="mask-group"></div>
+
+
+                </section>
+                <section class=" cardworks cardwork-five">
+                    <div class="action">
+                        
+                        <button type="button" disabled class="btn"><small class="btntxt">See Project</small></button>
+                        <hr class="rectangle">
+                    </div>
+                    <article class="cardwork-one-rb">
+                        <p class="cwo-title-post">Profesional Art <br>Printing Data</p>
+                        <p class="cwo-supporting-text">A daily selection of privately <br>
+                            personalized reads; no accounts or <br>
+                            sign-ups required. has been the <br>
+                            industry's standard</p>
+                        <ul class="cwo-tags">
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
+                        
+                    </article>  
+                    <div class="mask-group"></div>
+
+
+                </section>
+                <section class=" cardworks cardwork-six">
+                    <div class="action">
+                        
+                        <button type="button" disabled class="btn"><small class="btntxt">See Project</small></button>
+                        <hr class="rectangle">
+                    </div>
+                    <article class="cardwork-one-rb">
+                        <p class="cwo-title-post">Profesional Art <br>Printing Data</p>
+                        <p class="cwo-supporting-text">A daily selection of privately <br>
+                            personalized reads; no accounts or <br>
+                            sign-ups required. has been the <br>
+                            industry's standard</p>
+                        <ul class="cwo-tags">
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
+                        
+                    </article>  
+                    <div class="mask-group"></div>
+
+
+                </section>
+            </div>
+
```

Comment:
- [ ] It would be better if you remove the `div` tag as you already cover the content with the `section` tag you can use the div tag inside the section for better semantic HTML.

Reply:
@roshan-bajgain  the div is a container for the cards. I have 6 cards (<section class=" cardworks cardwork-one"> .... <section class=" cardworks cardwork-six">) and I contained them with div class. So I can't remove it. And I used CSS grid property on this (to put the section part in order).